types/secret-box/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for secret-box 0.1
+// Project: https://github.com/jprichardson/secret-box
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { BinaryLike } from 'node:crypto';
+
+/**
+ * Encrypt secrets. Built on AES-256-GCM and Scrypt.
+ *
+ * @example
+ * import * as secretBox from 'secret-box'
+ *
+ * const passphrase = Buffer.from('open sesame 2')
+ * const message = Buffer.from('The secret launch code is 1234.')
+ *
+ * const secret = secretBox.encrypt(message, passphrase)
+ */
+export function encrypt(message: Buffer, passphrase: string | Buffer, opts?: EncryptOptions): Buffer;
+/**
+ * Decrypt secrets. Built on AES-256-GCM and Scrypt.
+ *
+ * @example
+ * import * as secretBox from 'secret-box'
+ *
+ * const passphrase = Buffer.from('open sesame 2')
+ * const message = Buffer.from('The secret launch code is 1234.')
+ *
+ * const secret = secretBox.encrypt(message, passphrase)
+ * const decrypted = secretBox.decrypt(secret, passphrase)
+ *
+ * console.log(decrypted.toString('utf8'))
+ * // => The secret launch code is 1234.
+ */
+export function decrypt(secret: Buffer, passphrase: string | Buffer): Buffer;
+export const struct: any;
```

Comment:
does it export an object with `decode`/`encode` methods? (serialize/deserialize)

Reply:
Yeah, just created the type for this. I'll wait until they're merged and will then rebase this branch.